.gitignore

```diff
@@ -47,23 +47,24 @@ buck-out/
 # Jest
 .jest/
 android/.settings/org.eclipse.buildship.core.prefs
-example/android/.settings/org.eclipse.buildship.core.prefs
+examples/purchaseTester/android/.settings/org.eclipse.buildship.core.prefs
+examples/MagicWeather/android/.settings/org.eclipse.buildship.core.prefs
 android/gradlew
 android/gradlew.bat
 android/gradle/wrapper/gradle-wrapper.jar
 android/gradle/wrapper/gradle-wrapper.properties
 .vscode/launch.json
 .vscode/settings.json
 android/.classpath
-android/.project
-example/.vscode/launch.json
-example/.vscode/settings.json
-example/android/app/.classpath
-example/android/app/.project
-example/android/app/.settings/org.eclipse.buildship.core.prefs
-example/.vscode/
+*.project
```

Comment:
I think this should be `**/*.project` to ignore it in any subfolder

Reply:
good call